chia/wallet/did_wallet/did_wallet.py

```diff
@@ -222,9 +226,11 @@ async def get_pending_change_balance(self) -> uint64:
 
         return uint64(addition_amount)
 
-    async def get_unconfirmed_balance(self, record_list=None) -> uint64:
-        confirmed = await self.get_confirmed_balance(record_list)
-        return await self.wallet_state_manager._get_unconfirmed_balance(self.id(), confirmed)
+    async def get_unconfirmed_balance(self, record_list=None) -> uint128:
+        # confirmed = await self.get_confirmed_balance(record_list)
+        # This seems like a bug - did we intend to also pass in other unspent CoinRecords?
+        # return await self.wallet_state_manager.get_unconfirmed_balance(self.id(), confirmed)
+        return await self.wallet_state_manager.get_unconfirmed_balance(self.id())
```

Comment:
@matt-o-how I'm wondering if the `confirmed` parameter is leftover code from an earlier time.

If not, we'll have to change this line to collect a set of `CoinRecord`s, instead of a count of mojos.

Reply:
This seems like a bug - did we intend to also pass in other unspent CoinRecords, or did we intend to move to just getting the total mojos?